VERSION

```diff
@@ -1 +1 @@
-2.3.0
```

Comment:
The VERSION change really must be a PR in itself (perhaps with documentation changes). 

Can you remove this change?